website/ADD_LOGO.md

```diff
@@ -0,0 +1,54 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+-->
+
+# How to add your logo
+
+1. Fork [Apache Beam](https://github.com/apache/beam) repository
+2. Add file with company or project name to the [case-studies](https://github.com/apache/beam/tree/master/website/www/site/content/en/case-studies) folder
+3. Add project/company logo to the [images](https://github.com/apache/beam/tree/master/website/www/site/static/images) folder
```

Comment:
@ElessarST reviewed and decided to use https://github.com/apache/beam/tree/master/website/www/site/static/images/logos/powered-by
Could you please change company logos to powered-by folder?